packages/mdsvex/src/globals.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module '*.svx' {
+	import type { SvelteComponentDev } from 'svelte/internal';
+
+	export default class Comp extends SvelteComponentDev {
+		$$prop_def: {};
```

Comment:
This could be simplified if the `svelte` dependency was higher, so I could use `SvelteComponentTyped` and I could get rid of $$prop_def

Reply:
Not a big issue though. Feel free to resolve